lte/gateway/c/oai/tasks/s1ap/s1ap_state.cpp

```diff
@@ -168,7 +168,7 @@ bool get_mme_ue_ids_no_imsi(
   hash_table_ts_t* s1ap_ue_state = get_s1ap_ue_state();
   hashtable_ts_get(s1ap_ue_state, (const hash_key_t) dataP, (void**) &ue_ref_p);
   if (!ue_ref_p) {
-    *mme_id_list[(*num_ues_checked)++] = keyP;
+    (*mme_id_list)[(*num_ues_checked)++] = keyP;
```

Comment:
stylewise it is more readable IMO to have the increment on a separate line.

Reply:
@ulaskozat , fixed in latest version